app/src/test/java/util/CheckRangeTest.java

```diff
@@ -0,0 +1,30 @@
+package util;
+
+import org.junit.Test;
+
+import swati4star.createpdf.util.PDFUtils;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class CheckRangeTest {
```

Comment:
Change the name of the file and the class to `PDFUtilsTest.java`

Reply:
okay